windu.el

```diff
@@ -71,6 +71,15 @@
             (define-key map (kbd "C-.") 'windu-bring-right)
             (define-key map (kbd "C-<") 'windu-bring-top)
             (define-key map (kbd "C->") 'windu-bring-bottom)
+            ;; Window Configuration
+            (define-key map (kbd "2") 'windu-set-window-configuration-2)
+            (define-key map (kbd "3") 'windu-set-window-configuration-3)
+            (define-key map (kbd "4") 'windu-set-window-configuration-4)
+            (define-key map (kbd "5") 'windu-set-window-configuration-5)
+            (define-key map (kbd "@") 'windu-load-window-configuration-2)
```

Comment:
Getting used to these still.

I keep wanting to do `C-x C-x C-2` instead of `C-x C-x Shift-2`.  Switching modifier keys mid-way (from control to shift) is hard!